src/app.ts

```diff
@@ -27,8 +26,6 @@ if (!process.env.PUBLIC_URI) {
 (async () => {
 
   console.log('Connecting to database');
-  await checkPatches();
```

Comment:
I think we need to auto-run knex migrations on start. I believe knex ensures that if multiple a12nserver start at the same time, only 1 will run the migration (using table locks).

Reply:
I think we need to migrate the migrations to knex.

I don't think this means a full-rewrite of all migrations to knex migrations, but I do think we need a knex migration that:

1. Checks the patch level
2. If it's below the latest patch (from the old patch system), either:
  * complain and die and tell users to manually apply these patches
  * Or: automatically run the patches (with `.raw` ?)

This is to support an upgrade path from an earlier a12nserver.

Reply:
Hmmm, I think I might go with the complain and die approach for now...

Reply:
Yeah i think that's fine. This will at least prevent people from starting the server with a broken database.